app/src/main/res/layout/fragment_profile_badges.xml

```diff
@@ -1,151 +1,141 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".ProfileBadges">
 
     <LinearLayout
-        android:layout_width="370dp"
-        android:layout_height="134dp"
-        android:layout_marginLeft="15dp"
-        android:layout_marginTop="200dp"
-        android:layout_marginRight="15dp"
-        android:layout_marginBottom="20dp"
+        android:id="@+id/first_badge"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginTop="32dp"
         android:background="@drawable/badge_round_background"
-        android:orientation="horizontal">
-
-        <LinearLayout
-            android:layout_width="142dp"
-            android:layout_height="match_parent"
-            android:layout_marginTop="15dp"
-            android:orientation="horizontal">
-
-            <ImageView
-                android:id="@+id/banevolent_donor"
-                android:layout_width="wrap_content"
-                android:layout_height="103dp"
-                android:layout_weight="1"
-                android:src="@drawable/benevolent_donor_badge" />
-        </LinearLayout>
+        android:orientation="horizontal"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent">
+
+        <ImageView
+            android:id="@+id/neighborhood_all_star"
+            android:layout_width="100dp"
+            android:layout_height="100dp"
+            android:layout_gravity="center_vertical"
+            android:layout_margin="8dp"
+            android:src="@drawable/neighborhood_all_star_badge" />
 
         <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:orientation="vertical">
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:orientation="vertical"
+            android:padding="8dp">
 
             <TextView
-                android:id="@+id/textView_benevolent_badge"
+                android:id="@+id/textView_neighborhood_badge"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginTop="15dp"
-                android:text="@string/benevolent_donor"
-                android:textStyle="bold" />
+                android:text="Neighborhood All-Star"
```

Comment:
This needs to be a string resource as it was before mainly for spanish localization
@string/benevolent_donor